### ISSUE TITLE
add tags to reflect PR state, status and potential ecosystem impact

### ISSUE THREAD

**Issue by pq** [dart-lang/linter#2817]
**TL;DR:** some changes can cause significant ecosystem impact and need to be vetted carefully before landing lest we block an SDK roll to Flutter and Google3.  It would be great to add some tags to make this process more transparent and better tracked.

See: https://github.com/dart-lang/linter/pull/2810

- [x] `set: core` - in the core Dart rule set
- [x] `set: recommended` - in the recommended Dart rule set
- [x] `set: flutter` - in the recommended Flutter rule set
- [x] `set: internal` - used by a team internally at Google
- [x] `set: none` - not in a defined rule set (e.g., core, recommended, flutter)

**Comment by asashour** [comment]
I would like to add two points:

- Can we have a description of the labels, e.g. what is the difference between `set: flutter` and `customer: flutter`? What is `internal` (does it mean it doesn't affect external systems), etc.?
- If a help is needed to test the PRs against existing open source code, e.g. Flutter, then external members can also help in this regard.

**Comment by pq** [comment]
Thanks @asashour.  I added descriptions to the set labels.

![image](https://user-images.githubusercontent.com/67586/127891103-c7c2576c-988d-4a7b-ac0a-4130ea8f2a65.png)

The customer labels reflect asks from specific teams.

As for help with testing, that'd be welcome.  The process is a bit cumbersome at the moment but we'd like to see it improved.  I hope to get some time for this in the coming month.  I'll kick off a doc sooner than later and will start with a tracking issue.

**Comment by pq** [comment]
We're at a good place with this one IMO.  Let's open fresh issues to track future work.
